addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/thing-types.xml

```diff
@@ -189,6 +209,11 @@
 		<label>Play URI</label>
 		<description>Plays a given URI</description>
 	</channel-type>
+    <channel-type id="netControl">
+        <item-type>String</item-type>
+        <label>Control</label>
+        <description>Control the USB/Net Menu, e.g. Up/Down/Select/Back/PageUp/PageDown/Select[0-9]</description>
+    </channel-type>
```

Comment:
Can you add the posible values for this channel as options:
	
`<state>
  <options>
	<option value="Up">Up</option>
           ...
        </options>
</state>
`